theme/templates/pages/menus/dropdown.html

```diff
@@ -1,55 +1,36 @@
-{% load i18n pages_tags mezzanine_tags %}
+{% load mezzanine_tags %}
 {% spaceless %}
-{% if page_branch_in_menu %}
 
-{% if branch_level == 0 %}
 <ul class="nav navbar-nav">
-    {% for page in page_branch %}
-    {% if not has_home and page.is_primary and forloop.first %}
-    <li class="{% if on_home %} active{% endif %}" id="dropdown-menu-home">
-        <a href="{% url "home" %}">{% trans "Home" %}</a>
+    <li {% if request.get_full_path == "/my-resources/" %}class="active"{% endif %} id="dropdown-menu-my-resources">
+        <a href="/my-resources/">My Resources</a>
     </li>
-    {% endif %}
-    {% if page.in_menu %}
-    <li class="{% if page.has_children_in_menu %}dropdown{% endif %}{% if page.is_current_or_ascendant %} active{% endif %}"
-        id="dropdown-menu-{{ page.html_id }}">
-        <a href="{{ page.get_absolute_url }}">
-            {{ page.title }}
-            {% if page.has_children_in_menu %}<span class="caret"></span>{% endif %}
-        </a>
-        {% if page.has_children_in_menu %}{% page_menu page %}{% endif %}
+    <li {% if request.get_full_path == "/search/" %}class="active"{% endif %} id="dropdown-menu-search">
+        <a href="/search/">Discover</a>
+    </li>
+    <li {% if request.get_full_path == "/collaborate/" %}class="active"{% endif %} id="dropdown-menu-collaborate">
```

Comment:
Is there a reason why we are adding Id attributes to these list items?

Reply:
I just used the same id attributes that were being generated by mezzanine.  I'm fine with removing them if they're not doing anything.